openhtf/util/validators.py

```diff
@@ -118,16 +118,18 @@ def __init__(self, minimum=None, maximum=None, type=None):
 
   @property
   def minimum(self):
-    return self._minimum
+    converter = self._type or _identity
```

Comment:
As elegant as the boolean short-circuiting trick is here, I think it's better to explicitly check for `None` instead of relying on truthiness as its proxy. For example, it's possible (though unlikely, I admit) that we'll get an instance (with a `__call__` override) for type that has `__bool__` and `__nonzero__` overrides.

Reply:
Fixed, PTAL.